.github/workflows/docker.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     steps:
       - id: set_matrix
         run: |
-          MATRIX_CONFIG=$(if [ "${{ github.event_name }}" == "pull_request" ]; then echo '["ci"]'; else echo '["dev", "lean", "py310", "websocket", "dockerize"]'; fi)
+          MATRIX_CONFIG=$(if [ "${{ github.event_name }}" == "pull_request" ]; then echo '["dev"]'; else echo '["dev", "lean", "py310", "websocket", "dockerize"]'; fi)
```

Comment:
this is a bit cryptic, but the short story is we built only the `ci` layer in PRs (and a wider list on `master` push events), and I decided that building `dev` makes much more sense. I thought `ci` was all-encompassing when I first set this matrix, but turns out it's just a small layer on top of lean. Testing `dev` captures the bulk of the dockerfile.